internal/compiler/widgets/common/menu-base.slint

```diff
@@ -123,6 +123,15 @@ export component MenuItemBase {
                 padding-left: root.padding-left;
                 padding-right: root.padding-right;
 
+                spacing: 10px;
```

Comment:
Not sure if this the right way to ensure there's spacing between the label and the icon :sweat: also, every menu item is now shifted to the right (which is most GUIs do, right?)